dashboard/new-dashboard/src/components/common/icons/computer.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
For what do we need custom icons? Please use https://heroicons.com instead.

Reply:
See usages of `@heroicons/vue` in the project.

Reply:
These icons are part of the design that Ekaterina Zhuvarova made, so they are kind of "approved".